Makefile

```diff
@@ -36,10 +36,17 @@ TESTFLAGS    :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# Static linking with glibc is a bad time. If we're building statically,
+# assume musl is installed and use that.
```

Comment:
OK, if the goal is to avoid false promotion of static builds, I could add a comment that this is an exotic configuration, or I could pull this out of the makefile and into a separate script; how does that sound? which would you prefer?

By the way, isn't hostname resolution etc unaffected since we're using `netgo`?